app/package.json

```diff
@@ -6,6 +6,7 @@
     "start": "node ./bin/www"
   },
   "dependencies": {
+    "bcrypt": "^1.0.1",
```

Comment:
This should be strict

```javascript
bcrypt: "1.0.1"
```

Should also update `npm-shrinkwrap.json` by running `npm shrinkwrap`.